fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -611,6 +611,11 @@ public int getParallelExecInstanceNum() {
         return parallelExecInstanceNum;
     }
 
+    public int getDefaultDOP() {
```

Comment:
suggest to move this function into BackendCoreStat. since it not related to SessionVariable.